.circleci/config.yml

```diff
@@ -27,8 +27,7 @@ mainBuildFilters: &mainBuildFilters
     branches:
       only:
         - develop
-        - fix-ci-deps
-        - issue-23843_electron_21_upgrade
+        - issue-7306
```

Comment:
My intent here is to run the full suite of tests - and build the binary - in the feature branch, and then PR into there to fix any issues discovered.